queries.sql

```diff
@@ -16,10 +16,51 @@ SELECT name, escape_attempts FROM animals
 WHERE weight_kg > 10.5;
 
 SELECT * FROM animals
-WHERE neutered = true
+WHERE neutered = true;
 
 SELECT * FROM animals
 WHERE name NOT IN ('Gabumon');
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
```

Comment:
- [ ] Please you missed this query, from the project requirement, Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that the change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction. :pray: